document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/mappers/ApiToAccountingPeriodMapper.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.mappers;
+
+import org.mapstruct.DecoratedWith;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+import org.springframework.web.context.annotation.RequestScope;
+import uk.gov.companieshouse.api.model.accounts.CompanyAccountsApi;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.period.Period;
+
+@RequestScope
+@Mapper(componentModel = "spring")
+@DecoratedWith(ApiToAccountingPeriodMapperDecorator.class)
+public interface ApiToAccountingPeriodMapper {
```

Comment:
would have kept this as ApiToPeriodMapper as it was stating that it is mapping from the api to the Period model.